wait-for-statuses.py

```diff
@@ -0,0 +1,36 @@
+import urllib.request
+import json
+import subprocess
+import time
+import os
+
+# We're limited to this number by GH Actions API
+# https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#list-jobs-for-a-workflow-run
+max_jobs=100
+
+status_url = "https://api.github.com/repos/" \
+         + os.environ['GITHUB_REPOSITORY'] \
+         + "/actions/runs/" \
+         + os.environ['GITHUB_RUN_ID'] \
+         + "/jobs" \
+         + "?per_page=" + str(max_jobs)
+
+numOfJobs = int(os.environ['NUM_OF_JOBS'])
+
+while(True):
+  time.sleep(60)
+  countCompleted = 0
+
+  with urllib.request.urlopen(status_url) as url:
+    data = json.loads(url.read().decode())
+    for j in data["jobs"]:
+      if(j["status"] == "completed"):
```

Comment:
we should end the script, failing, whenever there's any failed status

Reply:
or cancelled, or anything that is not about to finish correctly

Reply:
I've added checks for "non-success" jobs and now we exit with error when any of jobs fails.